functions/private/Test-WinUtilPackageManager.ps1

```diff
@@ -20,13 +20,15 @@ function Test-WinUtilPackageManager {
     $status = "not-installed"
 
     if ($winget) {
-        # Install Winget if not detected
-        $wingetExists = Get-Command -Name winget -ErrorAction SilentlyContinue
+        # Get Winget's Version and Install Winget if it isn't detected
```

Comment:
```suggestion
        # Get Winget's Version
```
Maybe I am way too pedantic, but, I think the comment was not entirely correct from the get-go. 
The function `Test-WinUtilPackageManager` only checks if winget/chocolatey are installed and prints / returns the result. The logic to install winget is handled outside of this function so the commend isn't really accurate of what is happening in the script.

Reply:
Actually felt a bit bad writing the comment because it seemed so inconsequential to fix a single line that didn't even change anything on how the code behaved ^^